ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -60,7 +62,7 @@
   private BlobStoreCompactor compactor;
   private PersistentIndex index;
   private BlobStoreStats blobStoreStats;
-  private boolean started;
+  private volatile boolean started;
```

Comment:
why volatile. getDiskSpaceRequirements() will be called only when the BlobStore has finished starting right